### ISSUE TITLE
Make folder structure configurable/flexible

### ISSUE THREAD

**Issue by bohning** [bohning/usdb_syncer#16]
Currently, the folder structure is hardcoded `Artist - Title\{usdb_id}`. The user may wants to have a different folder structure, e.g. have a top level folder for every beginning letter of the Artist, or for every language or combinations of that. If we make this configurable, the sync process has to respect that directory structure.

**Comment by Baklap4** [comment]
Really would love this feature!
Personally i store everything like this:
```
karaoke
|---->USDB-live
|------> English
|---------> ABBA
|-------------> ABBA - Mamma mia
|----------------> ABBA - Mamma mia.mp3, ABBA - Mamma mia.mp4, ABBA - Mamma mia.txt
|------> Dutch
|---------> Youp van 't Hek
|-------------> Youp van 't Hek - Flappie
|----------------> Youp van 't Hek - Flappie.mp3, Youp van 't Hek - Flappie.mp4, Youp van 't Hek - Flappie.txt
```

But making it configurable would make even more sense :)

**Comment by bohning** [comment]
Unfortunately not. It's on the roadmap, but tbh with a low priority.

**Comment by bohning** [comment]
But it is already possible to reorganize your collection as you like, as long as you restart the syncer. On restart, the syncer will keep track of where to find which song.

**Comment by RumovZ** [comment]
Closed with #270.

**Comment by Baklap4** [comment]
Thanks going to check it out somewhere next week :D!
